tutor/resources/styles/variables/book-content.less

```diff
@@ -60,6 +61,11 @@
 @openstax-book-concepts_biology-bg: @openstax-book-orange;
 @openstax-book-concepts_biology-accent: @openstax-book-orange-accent;
 
+// History
+@openstax-book-ap_us_history-fg: rgb(241, 118, 72);
+@openstax-book-ap_us_history-bg: rgb(35,  48,  99);
```

Comment:
yeah, I thought that also - but wasn't sure and figured we should adhere to what the book used unless @Fredasaurus said otherwise